src/einsteinpy/plotting/geodesics_scatter.py

```diff
@@ -114,3 +133,6 @@ def show(self):
 
     def save(self, name="scatter_geodesic.png"):
         plt.savefig(name)
+
+    def get_swcradius(self):
```

Comment:
I think we don't need to add a redundant function

Reply:
Won't this be necessary for manual scaling? To get the multiplier.

I thought this function should be there for the geodesicplotterclass object otherwise the user will have to call it separately.
Will remove this if you say so.

Reply:
We already have schwarzschild radius in the utility. We don't need this redundant function

Reply:
Okay, removed this.
Done